modules/ROOT/pages/preview-community.adoc

```diff
@@ -0,0 +1,19 @@
+= Preview Your API Community
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+To show a live preview of the community's site, including APIs in *Preview Mode*, open the ACM administrator interface and select *Community Builder*.
+
+image::builder.png[Site Builder]
+
+Select *Preview*.
+
+To return to the site builder, select *Back to Builder*.
```

Comment:
This should probably be the final step in the preview task.

What comes next should be a separate topic, because it is not part of "Preview Your API Community."

Or, you could retitle this whole thing to something like Preview, Publish, and View Your Community.

Reply:
Updated to split into preview and publish task topics.